### PR TITLE
Issue 503/pill buttons

### LINKED ISSUE #503: Enhancement: Button consistency to pill shape
### Describe the Current Behavior/Feature:

Design-wise, the UX team was moving forward with buttons being "pill"-shaped. There are certain places where buttons remains rectangular boxes, such as the buttons within the user profile and the sign-in button.

### Rationale:

Keep things consistent with existing designs for the website.

### Proposed Implementation (if applicable):

The existing `border-radius` for most buttons utilizes a value of `20px`. Think we could have this implemented for any non-compliant button.

### PR DESCRIPTION
**Pull request by xscottxbrownx**
## This PR:
Resolves #503 
 
**1.**  Makes buttons consistent with pill shapes by editing in `theme.js`
**2.** Fixes some typos in `theme.js`
**3.** Clears unused code in `MessageStyles.js` and renamed to `PaginationStyles.js`

### CODE REVIEW

**Comment by xscottxbrownx** [comment]
Home test will be fixed in PR #516

File: src/components/Messages/MessageButtonGroup.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const MessageButtonGroup = ({ showModal, setShowModal, boxType, setBoxType }) =>
         onClick={() => setShowModal(!showModal)}
         startIcon={<CreateIcon />}
         color="secondary"
-        sx={{ width: isReallySmallScreen ? '200px' : 'default' }}
+        sx={{ borderRadius: '25px' }}
```
Comment (andycwilliams) [inline src/components/Messages/MessageButtonGroup.jsx]: Shouldn't this be 20 as well?

Comment (xscottxbrownx) [reply src/components/Messages/MessageButtonGroup.jsx]: I started it at 20, and it didn't look correct. test it out. I feel the extra 5px did the trick.

Comment (leekahung) [reply src/components/Messages/MessageButtonGroup.jsx]: Had just took a look between 20px and 25px for the smaller buttons.

Visually, I think we could have the smaller buttons use 25px as well, there's not much of a difference. We could get that `border-radius` standardized for all our buttons.

Comment (xscottxbrownx) [reply src/components/Messages/MessageButtonGroup.jsx]: > Had just took a look between 20px and 25px for the smaller buttons.
> 
> Visually, I think we could have the smaller buttons use 25px as well, there's not much of a difference. We could get that `border-radius` standardized for all our buttons.

Fair enough, switched all to 25px...

---

Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
`src > components > Messages > MessageStyles.js`

**Comment by xscottxbrownx** [comment]
@andycwilliams @leekahung 

So since we are using this on every single MUI Button component, I just moved it into `theme.js`.   So after this, anytime we put in a `<Button>` anywhere it will automatically have a `border-radius: 25px`, unless we specifically tell it differently via the `sx` prop.

----

But also new issue:

I believe only `PaginationContainer` is being used in the MessageStyles file (src > components > Messages > MessageStyles.jsx).   Just need a few other eyes to verify that before deleting it all and renaming to `PaginationStyles.jsx`.

**Comment by leekahung** [comment]
> @andycwilliams @leekahung 
> 
> So since we are using this on every single MUI Button component, I just moved it into `theme.js`.   So after this, anytime we put in a `<Button>` anywhere it will automatically have a `border-radius: 25px`, unless we specifically tell it differently via the `sx` prop.

Theming it would be good

File: src/components/Messages/MessageButtonGroup.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Messages/MessageButtonGroup.jsx]: > Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
> `src > components > Messages > MessageStyles.js`

I need to get a more thorough check later, but it does look like we moved over most of the styles to MUI minus the pagination container.

Think a clean up would be good.

**Comment by andycwilliams** [comment]
> @andycwilliams @leekahung
> 
> So since we are using this on every single MUI Button component, I just moved it into `theme.js`. So after this, anytime we put in a `<Button>` anywhere it will automatically have a `border-radius: 25px`, unless we specifically tell it differently via the `sx` prop.
> 
> But also new issue:
> 
> I believe only `PaginationContainer` is being used in the MessageStyles file (src > components > Messages > MessageStyles.jsx). Just need a few other eyes to verify that before deleting it all and renaming to `PaginationStyles.jsx`.

Putting it in the theme is a good call. 

I was wondering if we wanted to switch to the MUI pagination component instead. I have no strong feelings about it myself. There's nothing wrong with the React version.

**Comment by andycwilliams** [comment]
They aren't caused by this PR but I am now noticing a couple of typos in `theme` on lines 36 and 69. `severieties` and `coolor`, namely.

File: src/components/Messages/MessageButtonGroup.jsx (hunk 1 above)
Comment (xscottxbrownx) [reply src/components/Messages/MessageButtonGroup.jsx]: > > Could you verify for me that nothing is being used in `MessageStyles.js` except for the `Pagination Container` styles?  We could really clean up that file if so.
> > `src > components > Messages > MessageStyles.js`
> 
> I need to get a more thorough check later, but it does look like we moved over most of the styles to MUI minus the pagination container.
> 
> Think a clean up would be good.

Feel like this should be done here in this PR?

**Comment by xscottxbrownx** [comment]
> They aren't caused by this PR but I am now noticing a couple of typos in `theme` on lines 36 and 69. `severieties` and `coolor`, namely.

fixed

File: src/components/Messages/MessageStyles.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const StyledButton = styled('button')({
   height: '60px',
   backgroundColor: '#017969',
   borderColor: 'black',
-  borderRadius: '5px',
+  borderRadius: '25px',
```
Comment (milofultz) [inline src/components/Messages/MessageStyles.js]: Why are we not using our MUI button that we are adding these default styles to?

https://github.com/codeforpdx/PASS/blob/8dd714b95c3748bc50545438e9347e0a363e38a5/src/theme.js#L58-L64

Comment (milofultz) [reply src/components/Messages/MessageStyles.js]: Is this styled button just a relic from another time? Just wondering if we can fold it all together so we don't have multiple same-but-different buttons

File: src/theme.js [hunk 3]
```diff
@@ -55,11 +55,18 @@ const theme = createTheme({
         }
       }
     },
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          borderRadius: 25
```
Comment (milofultz) [inline src/theme.js:61]: 🧠

**Review by milofultz — APPROVED** [review]
Just one q on styled button.

File: src/components/Messages/MessageStyles.js (hunk 2 above)
Comment (milofultz) [reply src/components/Messages/MessageStyles.js]: If not in this PR, maybe a separate issue to keep things clean

Comment (leekahung) [reply src/components/Messages/MessageStyles.js]: > Is this styled button just a relic from another time? Just wondering if we can fold it all together so we don't have multiple same-but-different buttons

Have just checked, outside of the `Pagination Container`, the rest of the styles are relics from an earlier styling scheme.

Think we could remove them here, but we could also open up a fresh PR just to have them removed

File: src/components/Messages/MessageButtonGroup.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Messages/MessageButtonGroup.jsx]: Think we could do that. Also think we could have it on a fresh PR to keep this separate. https://github.com/codeforpdx/PASS/pull/540#discussion_r1405295269

File: src/components/Messages/MessageStyles.js (hunk 2 above)
Comment (xscottxbrownx) [reply src/components/Messages/MessageStyles.js]: cleared code that is no longer used and renamed file to `PaginationStyles.js`

File: src/components/Messages/MessageButtonGroup.jsx (hunk 1 above)
Comment (xscottxbrownx) [reply src/components/Messages/MessageButtonGroup.jsx]: Cleared unused code.

**Review by leekahung — APPROVED** [review]
I'm good with this and the cleanup. 👍 Approved

**Comment by leekahung** [comment]
As for the snapshot test, think we could leave it alone here. Once PR #516 gets merged, we should be able to update it with the new tests there.

**Pull request merged**